tests/log_fidelity/test_bgp_shutdown.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import pytest
+
+from tests.common.plugins.loganalyzer.loganalyzer import LogAnalyzer, LogAnalyzerError
+
+logger=logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('any')
+]
+
+expected_log = "admin state is set to 'down'"
+defined_action = "sudo config bgp shutdown all"
+
+def test_bgp_shutdown(duthosts, rand_one_dut_hostname):
+    duthost=duthosts[rand_one_dut_hostname]
+    loganalyzer = LogAnalyzer(ansible_host=duthost, marker_prefix="bgp_shutdown")
+    loganalyzer.expect_regex=[]
+
+    try:
+        loganalyzer.expect_regex.append(expected_log)
+        marker=loganalyzer.init()
+        duthost.command(defined_action)
+        logger.info("check for expected log in syslog")
+        loganalyzer.analyze(marker)
+
+    except LogAnalyzerError as err:
```

Comment:
I would also add a `finally` clause to run `bgp startup all` so that we leave the device in a good state after the test. @yxieca do you think it also makes sense to test for the correct log message for the `startup` event as well?